### ISSUE TITLE
README edit for Markdown Syntax

### ISSUE THREAD

**Issue by geophpherie** [hackoregon/2019-backend-cookiecutter-django#16]
Minor issue. Rendering of the Python Markdown code is slightly off.

Instead of:
<pre>
Add subpackages to your INSTALLED_APPS:

`` ` python INSTALLED_APPS = ( ... '{{ cookiecutter.python_subpackage }}', ... ) ```

Add {{ cookiecutter.python_package_namespace }}'s URL patterns:

`` ` python from {{ cookiecutter.python_package_namespace }}.{{ cookiecutter.python_subpackage }} import urls as {{ cookiecutter.python_subpackage }}_urls urlpatterns = [ ... url(r'^', include({{ cookiecutter.python_subpackage }}_urls)), ... ] ```
</pre>

It should be (fix space in the backticks and break into lines):
<pre>
Add subpackages to your INSTALLED_APPS:

``` python
INSTALLED_APPS = ( ... '{{ cookiecutter.python_subpackage }}', ... ) 
```

Add {{ cookiecutter.python_package_namespace }}'s URL patterns:

``` python 
from {{ cookiecutter.python_package_namespace }}.{{ cookiecutter.python_subpackage }} 
import urls as {{ cookiecutter.python_subpackage }}_urls 
urlpatterns = [ ... url(r'^', include({{ cookiecutter.python_subpackage }}_urls)), ... ] 
```
</pre>

**Comment by geophpherie** [comment]
Additionally, while addressing this issue, one could also remove the line break in-between "Non-Hack" and "Oregon Workflow" in the heading immediately preceding the section.

**Comment by bhgrant8** [comment]
Fixed in #29, please reopen if you see further issues
